ChargerMail/src/User.java

```diff
@@ -0,0 +1,87 @@
+
+/**
+ * Class to implement a user within email client system
+ * @author tevonwalker
+ *
+ */
+
+public class User {
+	
+	/**
+	 * Constructs user object and sets username to username. Also sets up the remote and local sites
+	 * @param username name to use for new user
+	 */
+	
+	public User(String username){
+		this.username = username;
+		localSite = new Site();
+		remoteSite = new Site();
+	}
+	
+	/**
+	 * Set user name 
```

Comment:
It doesn't matter to me. If Dr. Li said that it is not need, then from here forth, we won't do it for common functions such as get/set/const.